src/content/docs/en/reference/image-service-reference.mdx

```diff
@@ -173,6 +173,8 @@ export type ImageTransform = {
     src: ImageMetadata | string;
     width?: number;
     height?: number;
+    widths?: number[] | undefined;
+	  densities?: (number | `${number}x`)[] | undefined;
```

Comment:
Looks like it breaks syntax highlighting

Reply:
GitHub seems to be the only one having a problem with it, it works both on the docs and in VS Code

Reply:
Feel free to resolve the conversation then 👍